### PR TITLE
recipe results styling

### LINKED ISSUE #31: Recipe results styling
**Is your feature request related to a problem? Please describe.**
The resulting recipes from the ingredients page need to be styled.

**Describe the solution you'd like**
Styled componentss

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Recipe card styling

## Purpose
Prettify the recipe cards

## Approach
Using styled components to dynamically render different layouts of the card.

Closes #31

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Nice job! No complaints here

**Review by JorgeLVilla — APPROVED** [review]
Nice job Paul!

File: src/components/MainContainer.tsx [hunk 1]
```diff
@@ -25,29 +25,32 @@ const Container = styled.div`
...
+    );
+    
+    // REVIEW FOR BEST WAY TO INCORPORATE DYNAMIC GRID
+    const BlobsContainer = styled.div`
+    grid-area: 1 / 2 / 2 / 3;
+    background-color: #ebe6da;
+    padding: 10px;
+    display: grid;
+    grid-template-columns: ${!userSearchedRecipes ? `1fr 1fr 1fr 1fr` : `1fr 1fr`};
+    grid-auto-rows: 250px;
```
Comment (krferrell) [inline src/components/MainContainer.tsx:49]: Nice. Always learning new things by viewing these PR's.

File: src/components/MainContainer.tsx [hunk 2]
```diff
@@ -25,29 +25,32 @@ const Container = styled.div`
...
+    grid-gap: 10px;
+    overflow: scroll;
```
Comment (krferrell) [inline src/components/MainContainer.tsx:51]: If you use overflow: auto, I believe the scrollbar is invisible until info actually overflows.

File: src/components/RecipeCard.tsx [hunk 3]
```diff
@@ -10,11 +14,84 @@ interface RecipeCardProps {
...
+`
+
+const AddBtn = styled.div`
+    width: 50px;
+    height: 50px;
+    background-color: #fff;
+    position: absolute;
+    top: 0;
+    left: 0;
+    border-radius: 15px 0 15px 0px;
```
Comment (krferrell) [inline src/components/RecipeCard.tsx:34]: extra px in the 0 at the end

File: src/components/RecipeCard.tsx [hunk 4]
```diff
@@ -10,11 +14,84 @@ interface RecipeCardProps {
...
+    const FoodImage = styled.div`
+        width: 50%;
+        background-color: #fff;
+        background-image: url(${image});
+        background-size: cover;
+        background-repeat: no-repeat;
+        height: 100%;
+        position: absolute;
+        right: 0;
+        clip-path: ellipse(50% 53% at 64% 29%);
```
Comment (krferrell) [inline src/components/RecipeCard.tsx:76]: Can't wait to see this

**Review by krferrell — APPROVED** [review]
Looking great!

File: src/components/MainContainer.tsx (hunk 2 above)
Comment (gonzalespaulb) [reply src/components/MainContainer.tsx:51]: Ill be trying that!

File: src/components/RecipeCard.tsx (hunk 3 above)
Comment (gonzalespaulb) [reply src/components/RecipeCard.tsx:34]: Good catch!

**Pull request merged**